### ISSUE TITLE
Dragged items in the collection are appearing on the admin page and the profile page.

### ISSUE THREAD

**Issue by jdaKoko** [Japan-Digital-Archives/Japan-Digital-Archive#922]
I created a collection, http://jdarchive.org/en/collection/1952783, which is not published.
The dragged items in my collection box are appearing in my profile page,http://jdarchive.org/en/user/421.
In addition, the dragged items,ID 1952784 ~1952802, are appearing on the admin dash page.  
Both phenomina crowd the profile page and the admin dashboard. Unwanted extra items on the admin dashboard slows down the curation job. I would be greatful if the IT team could fix it.  Thank you!

![collection npo](https://cloud.githubusercontent.com/assets/8831502/6087631/fbdbdaf2-ae1b-11e4-9651-ea705f5191c0.png)

**Comment by corinnecurcie** [comment]
So these items existed on our site before, but when you dragged them to the collection, it duplicated them in our database? Very strange! I will look into this.

**Comment by jdaKoko** [comment]
Yes your understanding is correct.  Thank you for looking into the problem. The duplicate seeds, which do not need to be exported, also show up on the export page.
